PrestaShop/PrestaShop
# Fixed link quick access

| Questions | Answers |
| --- | --- |
| Branch? | test |
| Description? | Fixed link quick access |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | See after |
- Check all links on sf controllers (product form or catalog)
- Check all links on legacy controllers
- Add a new link in sf controller
- Add a new link in legacy controller
- See if new links are ok on sf & legacy controllers

Suite of https://github.com/PrestaShop/PrestaShop/pull/6584
